packages/kit/types/app.d.ts

```diff
@@ -1,11 +1,21 @@
 import { ReadOnlyFormData, RequestHeaders } from './helper';
 import { ServerResponse } from './hooks';
+import { PrerenderOptions, SSRNodeLoader, SSRRoute } from './internal';
 
-export interface App {
-	init(): void;
+export class App {
+	constructor(manifest: SSRManifest);
 	render(incoming: IncomingRequest): Promise<ServerResponse>;
 }
 
+export class InternalApp extends App {
```

Comment:
perhaps this should be in `internal.d.ts`?

Reply:
I don't think it matters. If something is re-exported from `types/index.d.ts`, it's public, otherwise it's internal. There are plenty of examples of other internal types that are exported from files that also contain public types (e.g. `types/config.d.ts`)